app/scripts/controllers/users/edit.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name flightNodeDemo.controller:UserEditController
+ * @description
+ * # UserEditController
+ * Controller for the edit user page.
+ */
+angular.module('flightNodeDemo')
+	.controller('UserEditController', ['$scope', '$http', '$log', '$location', '$routeParams', 'messenger',
+		function ($scope, $http, $log, $location, $routeParams, messenger) {
+
+			$scope.loading = true;
+
+			var userId = $routeParams.userId;
+			if (!isFinite(userId)) {
+				// garbage input
+				return;
+			}
+
+			$scope.action = 'Edit';
+
+			var jwt = JSON.parse(localStorage.getItem('jwt'));
+
+			if (jwt) {
+				// TODO: validate that the user administration claim is present.
+				if (moment() < moment(jwt.expiresAt)) {
+
+					$http({
+						url: 'http://localhost:50323/api/v1/user/' + userId,
+						method: 'GET',
+						headers: {
+							Authorization: 'bearer ' + jwt.access_token
+						}
+					})
+						.then(function success(response) {
+
+							$scope.user = response.data;
+
+						}, function error(response) {
+
+							$log.error(response);
+
+							if (response.status === 401) {
+								messenger.unauthorized($scope);
+							} else {
+								messenger.showErrorMessage($scope, { error: response });
+							}
+						});
+				} else {
+					messenger.unauthorized($scope);
+				}
+			} else {
+				messenger.unauthorized($scope);
+			}
+
+			$scope.loading = false;
+
+			$scope.cancel = function () {
+				$location.path('/#/users');
+			}
+
+			$scope.submit = function () {
+				var jwt = JSON.parse(localStorage.getItem('jwt'));
+				
+				$log.info($scope);
+				
+				if (jwt) {
```

Comment:
too much code duplication around jwt handling. I need to resolve that by introducing a new service / factory.